manipulation/src/main/java/org/jboss/gm/manipulation/ManipulationPlugin.java

```diff
@@ -16,11 +17,12 @@
 
     @Override
     public void apply(Project project) {
-        final String projectName = project.getName();
+        // apply the maven publishing plugin
+        project.getPluginManager().apply(MavenPublishPlugin.class);
```

Comment:
What happens if the project already applies the mavenpublishplugin - do any gradle projects, expecting to be deployed, *not* put it in?

Reply:
There are projects that use the older `maven` plugin so we will need to take care of that case as well.

Also, applying pluggins is idempotend according to: https://docs.gradle.org/current/userguide/plugins.html#sec:using_plugins

Reply:
Ok but what happens if there is both maven and maven publish plugins?

Reply:
I haven't checked that use case yet.

Reply:
Me neither

Reply:
Ok, no problem, we need to log these thoughts in jira I think